### PR TITLE
SRCX-IoTG commit ARTIK IDE online installation package source code, please review.

### PR DESCRIPTION
**Pull request by lvbin0127**
SRCX-IoTG commit ARTIK IDE online installation package source code, please review.

### CODE REVIEW

**Comment by laralijuan** [comment]
@vparfonov could you help us review this PR? thank you~

File: plugins/plugin-artik-ide/src/main/java/org/eclipse/che/plugin/artik/ide/installpkg/PackageInstallerViewImpl.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+ *******************************************************************************/
+package org.eclipse.che.plugin.artik.ide.installpkg;
+
+import com.google.gwt.event.dom.client.ChangeEvent;
+import com.google.gwt.event.dom.client.ChangeHandler;
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.ui.*;
```
Comment (vparfonov) [inline plugins/plugin-artik-ide/src/main/java/org/eclipse/che/plugin/artik/ide/installpkg/PackageInstallerViewImpl.java]: avoid use "*" import

File: plugins/plugin-artik-ide/src/main/java/org/eclipse/che/plugin/artik/ide/installpkg/PackageInstallerViewImpl.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import com.google.inject.Inject;
+import org.eclipse.che.ide.api.machine.WsAgentURLModifier;
+import org.eclipse.che.ide.ui.window.Window;
+import org.eclipse.che.plugin.artik.ide.ArtikLocalizationConstant;
+//import org.eclipse.che.ide.upload.file.PackageInstallerView;
```
Comment (vparfonov) [inline plugins/plugin-artik-ide/src/main/java/org/eclipse/che/plugin/artik/ide/installpkg/PackageInstallerViewImpl.java]: remove commented code

**Comment by lvbin0127** [comment]
@vparfonov I have commit the modified source code based on Codenvy review comments, number is : 80c113452d90fa3aae1e5e06c3c53f1986739c7f, please review code, but i don't know how to create a new pull request, please give me some comments, thanks.

**Pull request merged**